### PR TITLE
Import Linux Kernel typos: letter R

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -9397,6 +9411,7 @@ recompence->recompense
 recomputaion->recomputation
 reconaissance->reconnaissance
 reconcilation->reconciliation
+reconfiged->reconfigured
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: reconfiged may be an intentional misspelling., I think we should either keep and add `reconfiged` or delete this entirely.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -9696,9 +9743,12 @@ reonly->read-only
 reopended->reopened
 reoport->report
 reord->record
+reorer->reorder
 reorganision->reorganisation
 reosurces->resources
 reounded->rounded
+reources->resources
+reourcesn->resourcesn
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: huh?

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Removed.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Removed both.

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: No need to remove the 1st one. The 2nd one was the issue.

**Comment by Gelma** [comment]
I close this and recreate a second versione to avoid conflicts.

**Pull request closed without merging**